website/source/docs/agent/configuration/vault.html.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: "docs"
+page_title: "vault Stanza - Agent Configuration"
+sidebar_current: "docs-agent-configuration-vault"
+description: |-
+  The "vault" stanza configures Nomad's integration with HashiCorp's Vault.
+  When configured, Nomad can create and distribute secrets to tasks
+  automatically.
+---
+
+# `vault` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>**vault**</code>
+    </td>
+  </tr>
+</table>
+
+
+The `vault` stanza configures Nomad's integration with
+[HashiCorp's Vault][vault]. When configured, Nomad can create and distribute
+secrets to tasks automatically. For more information on the architecture and
+setup, please see the [Nomad and Vault integration documentation][nomad-vault].
+
+```hcl
+vault {
+  enabled = true
+  address = "https://vault.company.internal:8200"
+}
+```
+
+## `vault` Parameters
+
+- `address` - `(string: "https://vault.service.consul:8200")` - Specifies the
+  address to the Vault server. This must include the protocol, host/ip, and port
+  given in the format `protocol://host:port`. If your Vault installation is
+  behind a load balancer, this should be the address of the load balancer.
+
+- `allow_unauthenticated` `(bool: false)` - Specifies if users submitting jobs
+  to the Nomad server should be required to provide their own Vault token,
+  proving they have access to the policies listed in the job. If enabled, users
```

Comment:
Last sentence change to: "This option should only ever be enabled in a trusted environment"